templates/service_page.hbs

```diff
@@ -14,13 +14,18 @@
   const settings = {{json settings}};
   const container = document.getElementById("service-catalog-item");
 
+  // Extract the ID from the URL
+  const url = window.location.href;
+  const id = url.substring(url.lastIndexOf('/') + 1);
```

Comment:
`window.location.href` will return query params as well if there's ever some and that might break this logic. 
A more resilient way would be `window.location.pathname.split("/").pop()`.
We might as well consider exposing it via curlybars at some point.